config.toml.example

```diff
@@ -523,7 +523,8 @@ changelog-seen = 1
 #test-compare-mode = false
 
 # Use LLVM libunwind as the implementation for Rust's unwinder.
-#llvm-libunwind = false
+# Accepted values are 'in-tree' or 'system'.
+#llvm-libunwind = 'no'
```

Comment:
Hm, so presumably `'no'` is also an accepted value? (mirroring false before)?

Could we make a note here that false = 'no' and true = 'in-tree' as well?

Reply:
Well, accepted values are `Some("in-tree")` and `Some("system")`, every other possibility is ignored, e.g. what `false` did indeed